docs/the-top-10/introduction_for_ot.md

```diff
@@ -0,0 +1,29 @@
+# The security World for OT: Why is it problematic?
+
+Malicious external actors were previously not a problem for the operators.  These dangers were not relevant. In addition, direct physical access was usually required to manipulate the system.
+
+OT operators are familiar with the production process. Cybersecurity is a new field in which there is little knowledge.
+
+## Problems with security
+
+Many cyber security measures make it difficult to analyze errors if there is a problem with the machine or system. For example, an encrypted transmission of data cannot easily be analyzed to determine whether the data is correct. 
+
+Another example is the use of certificates for authentication. If these expire and are not replaced in time, this can lead to the system coming to a standstill. This leads to a reduction in the availability of the system. 
+
+There are therefore some concerns regarding the use of cyber security measures.
+
+## Poor usability of cybersecurity features
+
+In many cases, the configuration of cyber security functions is very complex and overwhelms the operators.
```

Comment:
maybe add: While "secure-by-default" has been established for newer IT products, i.e., the default configuration is mostly secure, this hasn't been the case for OT yet. This means that configuring security is often an additional manual step which is problematic, esp. if deadlines are coming nearer.